script/DeployYnEigenViewer.s.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: BSD 3-Clause License
+pragma solidity ^0.8.24;
+
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import {ynEigenViewer} from "../src/ynEIGEN/ynEigenViewer.sol";
+
+import "./BaseYnEigenScript.s.sol";
+
+contract DeployYnEigenViewer is BaseScript {
+
+    ynEigenViewer public viewer;
+
+    address public constant ASSET_REGISTRY = 0xc92b41c727AC4bB5D64D9a3CC5541Ef7113578b0;
```

Comment:
Was thinking to add these to ContractAddresses already.

We have the

    struct YieldNestAddresses {
        address YNETH_ADDRESS;
        address STAKING_NODES_MANAGER_ADDRESS;
        address REWARDS_DISTRIBUTOR_ADDRESS;
        address EXECUTION_LAYER_RECEIVER_ADDRESS;
        address CONSENSUS_LAYER_RECEIVER_ADDRESS;  
    }
    
    Struct.
    
    Keeping in mind we might add ynBTCe, ynUSDe etc.
    
    And they all have the same contracts deployed - 
    
    Could it be that we add something like a mapping to YieldNest contracts that maps an enum to another struct that has these defined?  (ynEigen, tokenStakingNodesManager, etc.)
    
    Feel free to do another structure depending on how you see it make sense.